app/views/auth_source_externals/_external_card.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="dropdown pull-right dropdown-kebab-pf">
+  <button class="btn btn-link dropdown-toggle" type="button" id="dropupKebabRight2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <span class="fa fa-ellipsis-v"></span>
+  </button>
+  <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropupKebabRight2">
```

Comment:
I would be happy if in the era of React and Redux we could create this code in webpack already,?
I know it might take some work but imo the code would be more readable, testable with all of the modern convention,
For any help me and the @theforeman/ui-ux are here to help.

The dropdown component from patternfly-react was used [here](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/bookmarks/index.js#L60).